mne/io/tests/test_pick.py

```diff
@@ -8,7 +8,8 @@
 
 from mne import (pick_channels_regexp, pick_types, Epochs,
                  read_forward_solution, rename_channels,
-                 pick_info, pick_channels, __file__, create_info)
+                 pick_info, pick_channels, create_info)
+from mne import __file__ as my_file
```

Comment:
the mne __file__ refers to `mne-python/mne/__init__.py` and pytest collector confuses this path with `mne/io/tests/test_pick.py` file. Which if I'm not mistaken, is the value of __file__ when evaluating test_pick.py.